tools/QuietBackgroundProcesses/DevHome.QuietBackgroundProcesses.UI/Views/ProcessPerformanceTableControl.xaml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="utf-8"?>
+<UserControl
+    x:Class="DevHome.QuietBackgroundProcesses.UI.Views.ProcessPerformanceTableControl"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:DevHome.QuietBackgroundProcesses.UI"
+    xmlns:ui="using:CommunityToolkit.WinUI">
+
+    <UserControl.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <ResourceDictionary>
+                    <Style x:Key="BorderStyle" TargetType="Border">
+                        <Setter Property="BorderBrush" Value="{ThemeResource DividerStrokeColorDefaultBrush}" />
+                    </Style>
+                </ResourceDictionary>
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+    </UserControl.Resources>
+
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition />
+        </Grid.RowDefinitions>
+
+        <ListView
+            ItemsSource="{x:Bind ItemsSource}"
+            SelectionMode="Extended"
+            HorizontalContentAlignment="Stretch"
+            Margin="0"
+            Padding="0"
+            ui:ListViewExtensions.AlternateColor="{StaticResource CardBackgroundFillColorDefaultBrush}">
+            <ListView.Header>
+                <Grid ColumnSpacing="0">
+                    <Grid.RowDefinitions>
+                        <RowDefinition Height="40"/>
```

Comment:
At some point you should go through this with System Text scaling up to 200%. You'll probably want to hard-code less and/or make some things that are currently Height/Width => MinHeight/MinWidth, to prevent text getting cut off and accessibility bugs.

Reply:
I got super stumped - everything started with * and Auto, but devolved to hardcoded values over days of fumbling around.  Need to learn the magics.

Reply:
We should definitely address in the next pass.

Because this isn't part of the main Dev Home window, but rather limited to the pop up dialog, going to suggest WONTFIX (and feel guilty about it)